rel/leo_manager/reltool.config.in

```diff
@@ -130,5 +130,7 @@
            {copy, "../../apps/leo_manager/snmp/snmpa_manager_1/agent/conf/context.conf",   "snmp/snmpa_manager_1/agent/conf/context.conf"},
            {copy, "../../apps/leo_manager/snmp/snmpa_manager_1/agent/conf/standard.conf",  "snmp/snmpa_manager_1/agent/conf/standard.conf"},
            {copy, "../../apps/leo_manager/snmp/snmpa_manager_1/agent/conf/vacm.conf",      "snmp/snmpa_manager_1/agent/conf/vacm.conf"},
-           {copy, "../../apps/leo_manager/snmp/snmpa_manager_1/agent/conf/notify.conf",      "snmp/snmpa_manager_1/agent/conf/notify.conf"}
+           {copy, "../../apps/leo_manager/snmp/snmpa_manager_1/agent/conf/notify.conf",      "snmp/snmpa_manager_1/agent/conf/notify.conf"},
+           %% snmp-related
```

Comment:
"snmp-related" should be "prometheus-related" as well as other reltool.config.in.